processes/UofM.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, *args):
         self.createS3Client()
 
     def runProcess(self):
-        with open('ingestJSONFiles/UofM_CSV.json') as f:
+        with open('ingestJSONFiles/UofM_One_PD.json') as f:
```

Comment:
Could we pass in the file name as an argument in run process?

Reply:
The way the processes in the codebase are formatted to run we can't pass the file name as an argument. I can explain more when we speak later this week.